### ISSUE TITLE
BB#??

### ISSUE THREAD

**Issue by Bbbolaget-karl** [sweco-seukni/Bbbolaget#53]
apelsinchokladporter/stout
pale ale
ljus chokladmalt
mörk chokladmalt
svartmalt?
brunmalt?
Admiral - för apelsinen
5,5%abv

edit: havre också, för att den ska bli mild och rund. Därför stout, fast det borde gå lika bra med havreporter?

**Comment by Bbbolaget-karl** [comment]
En recap på tidigare Sjöbusar också den som vi kallar Radargruppcentral Myran (BB#21) och BB#10 Belgian pale ale

BB#10
Pilsner 5,1
Caramunich 0,34
Bisquit 0,13
Carapils 0,2
mäsktemp 67, 1h, koktid 1 h
18IBU ekg @60,10 g Taget@10, 9 g ekg @0
OG1062FG1005, troligen wlp545, abv7,3%

BB#21 Radargruppcentral Myran
Pilsner 4,6
Caramunich 0,3
Bisquit 0,1
Carapils 0,2
Mäkstemp 67, 1h, koktid 90 min
18IBU ekg @60,10 g ekg@10, 9 g ekg @0
OG1068FG1012, wlp515, abv 7,6%

BB

**Comment by Bbbolaget-karl** [comment]
BB#32 Sjöbusen ver.1 (recept fr Radical Brewing, s.117)
Pale Ale 1,8
Munich I 1,4
Amber 0,45
CaraMünich 0,17
Oraff socker 0,68
Mäsktemp 67, 1 h, koktid 1h, 30 IBU
24 IBU ekg @60, 6 IBU saaz@15, 20g saaz@0, socker i @5
Spädde till OG1050. FG1004, 6,0%abv, wlp545

BB#39 a&b
Samma recept som #32 fast med 630 g socker i. Socker i efter högkröjs istället. 
OG 1050 (enligt recept 1055, gissar att sockret är inräknat där)
WLP 545 FG 1006
WLP 550 FG 1012

**Comment by sweco-seukni** [comment]
Finns det något tema på denna tråden? 😀

Jag gillar idén med apelsinchocklad portern, den jag drack på Tröls smakade verkligen chocklad (chockladglass, chockladmousse), hur får man till det? Kanske kan skippa både brun och svartmalt så kanske chockladmalten gör jobbet

**Comment by Bbbolaget-karl** [comment]
Ja, temat är (potentiella)  framtida bryggningar. Gör lite research och
vill inte glömma bort det jag hittar.

Kul det du säger. Låter bra. Special B också?  Har inventerat och slängt
lite här hemma också.
Den 15 jan 2016 08:27 skrev "sweco-seukni" <notifications@github.com>:

**Comment by Bbbolaget-karl** [comment]
En havrestout, enligt recept i brewing classic styles

**Comment by Bbbolaget-karl** [comment]
gamla Flygbunkern version 3. (Version 2 har samma maltnota)

5,6 kg pilsnermalt
0,74 kg havre
0,28 kg Crystal 40
0,28 kg Crystal 80
0,23 kg Chokladmalt
0,34 kg Rostat korn

mäsks vid 67°C. 
OG 1074
45 IBU ekg vid 60 min
koka 90 min, gissar jag. 
wlp007 - dry english ale

**Comment by Bbbolaget-karl** [comment]
Här kan vi ju bocka av både sjöbusen och havrestout. Skoj. 

Bidrar med en ny,  från brew your own, trappist single, kallar de den.  

Stil:
Og:1044-1054
Fg:1004-1010
Ibu:25-45
Abv:4,8-6,0%

Deras
Og 1049 fg 1007 ibu 35 abv 5,5%

4.5 kg belgisk pilsnermalt
57 g styrkan golding, 4,5%alfa vid 60min
57 g saaz vid 0min
Wlp 530
Mäska lågt. De kör ett steg vid 55 grader och sen ett vid 63, innan utmäskning. 
90 min kok. 
Jässtart vid 18 men låterjästen

**Comment by Bbbolaget-karl** [comment]
Forts...
Låter jästen och temperaturen stiga fritt. 
Kolsyresätt till 3 volymer

**Comment by Bbbolaget-karl** [comment]
På 4,5 kg malt får vi ut mer än og 1049,  särskiljt vid 90 min kok.  

Sen tänker jag också att det kanske hade varit bättre med lägre og, säg 1046.

**Comment by Bbbolaget-karl** [comment]
De säger också att socker är onödigt i den här ölen...

**Comment by Bbbolaget-karl** [comment]
Missade en kort rast på 70 också. 
Hela ser ut såhär 38/55/63/70/78 med tiderna 
10/10/40/10/10

**Comment by Bbbolaget-karl** [comment]
Mer kommentarer. De säger eg wyeast 3787, vilket är den som wertvleteren och westmalle använder. Men också att ifall man försöker hålla nere dem här temperaturen, gör inte det
